import-export-cli/integration/devFirst_test.go

```diff
@@ -122,100 +126,131 @@ func TestInitializeAPIFromAPIDefinitionURL(t *testing.T) {
+		srcAPIM:   apim,
+		initFlag:  projectName,
+		oasFlag:   utils.TestOpenAPI3DefinitionPath,
+		forceFlag: false,
+	}
 
-	result, error := importApiFromProject(t, projectName, apim.GetEnvName())
-	assert.Nil(t, error, "Error while generating Project")
-	assert.Contains(t, result, "Successfully imported API", "Test InitializeProjectWithDefinitionFlag Failed")
+	//Assert that project import to publisher portal is successful
+	validateImportInitializedProject(t, args)
 
 	//Remove Created project and logout
 	base.RemoveDir(projectName)
-	t.Cleanup(func() {
-		base.Execute(t, "logout", apim.GetEnvName())
-	})
 }
 
 //Import API from initialized project from API definition which is already in publisher without --update flag
 func TestImportProjectCreatedFailWhenAPIIsExisted(t *testing.T) {
 	apim := apimClients[0]
 	projectName := "OpenAPI3Project"
+	username := superAdminUser
+	password := superAdminPassword
 
-	InitializeAPIFromDefinition(t, projectName, utils.TestOpenAPI3DefinitionPath)
+	args := &initTestArgs{
+		ctlUser:   credentials{username: username, password: password},
+		srcAPIM:   apim,
+		initFlag:  projectName,
+		oasFlag:   utils.TestOpenAPI3DefinitionPath,
+		forceFlag: false,
+	}
 
 	//Import API for the First time
-	importApiFromProject(t, projectName, apim.GetEnvName())
+	validateImportInitializedProject(t, args)
 
+	time.Sleep(1 * time.Second)
 	//Import API for the second time
-	result, _ := importApiFromProject(t, projectName, apim.GetEnvName())
-	assert.Contains(t, result, "Resource Already Exists", "Test InitializeProjectWithDefinitionFlag Failed")
+	validateImportFailedWithInitializedProject(t, args)
 
 	//Remove Created project and logout
 	base.RemoveDir(projectName)
-	t.Cleanup(func() {
-		base.Execute(t, "logout", apim.GetEnvName())
-	})
 }
 
 //Import API from initialized project from API definition which is already in publisher with --update flag
 func TestImportProjectCreatedPassWhenAPIIsExisted(t *testing.T) {
 	apim := apimClients[0]
 	projectName := "OpenAPI3Project"
+	username := superAdminUser
+	password := superAdminPassword
 
-	InitializeAPIFromDefinition(t, projectName, utils.TestOpenAPI3DefinitionPath)
+	args := &initTestArgs{
+		ctlUser:   credentials{username: username, password: password},
+		srcAPIM:   apim,
+		initFlag:  projectName,
+		oasFlag:   utils.TestOpenAPI3DefinitionPath,
+		forceFlag: false,
+	}
 
 	//Import API for the First time
-	importApiFromProject(t, projectName, apim.GetEnvName())
+	validateImportInitializedProject(t, args)
 
+	time.Sleep(1 * time.Second)
 	//Import API for the second time
-	result, error := importApiFromProjectWithUpdate(t, projectName, apim.GetEnvName())
-	assert.Nil(t, error, "Error while generating Project")
-	assert.Contains(t, result, "Successfully imported API", "Test InitializeProjectWithDefinitionFlag Failed")
+	validateImportUpdatePassedWithInitializedProject(t, args)
 
 	//Remove Created project and logout
 	base.RemoveDir(projectName)
```

Comment:
Can we move this _base.RemoveDir()_  into the _validateImportInitializedProject()_ function and register it as a t.Cleanup()? It is an essential cleanup step when running this scenario but it is a low level detail that is not required to be known at the high level of the test case.